app/controllers/notifications_controller.rb

```diff
@@ -26,33 +31,14 @@ def fetch_notifications
   end
 
   def signed_in
-    if !user_signed_in?
-      signed_in = -1
-    else
-      signed_in = current_user.id
-    end
-    result = { signed_in: signed_in }
+    result = { signed_in: current_user.id }
     respond_to do |format|
       format.html { render json: result }
       format.json { render json: result }
     end
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_notification
-      begin
-        @notification = Notification.find(params[:id])
```

Comment:
This used to let users destroy notifications belonging to other users provided that they knew (or guessed) a valid Notification ID.  I got rid of this `before_action` since it was only used in one place and was duplicating logic and used a more specific query in in `#destroy`.